src/api.js

```diff
@@ -0,0 +1,44 @@
+const API_KEY = 'Xc9LpWvBxMXHyr3QJGsT';
+const API_ENDPOINT = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${API_KEY}/scores`;
+
+// Function to save player score to the API
+export async function saveScoreToAPI(playerData) {
+  try {
+    const response = await fetch(API_ENDPOINT, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(playerData),
+    });
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    return response.json();
+  } catch (error) {
+    throw new Error(`Error saving player score: ${error.message}`);
+  }
+}
```

Comment:
- [ ] Ouch! this is close but as part of the project requirements you should use arrow functions instead of the function keyword whenever you create functions so please replace all occurrences of the functionn definition with an arrow function expression instead 
- [ ] Here's the project requirements as a reminder:
  ![firefox_0tZqmc8NvB](https://github.com/Danny-imo/Leaderboard/assets/104968236/b0beb264-1278-45f6-86b1-7a8982d4619f)